docsets/Werkzeug/docset.json

```diff
@@ -1,10 +1,22 @@
 {
     "name": "Werkzeug",
-    "version": "0.9.6",
+    "version": "0.16.1",
     "archive": "Werkzeug.tgz",
     "author": {
-        "name": "Byron Yi",
-        "link": "https://github.com/byronyi"
+        "name": "Torin Kwok",
+        "link": "https://github.com/torinkwok"
     },
-    "aliases": []
+    "aliases": [
+        "werkzeug"
+    ],
+    "specific_versions": [
+        {
+            "version": "0.16.1",
+            "archive": "versions/0.16.1/Werkzeug.tgz"
+        },
+        {
+            "version": "0.9.6",
+            "archive": "versions/0.9.6/Werkzeug.tgz"
```

Comment:
You need to add this archive as well.

Reply:
Done. https://github.com/Kapeli/Dash-User-Contributions/pull/2675/commits/ed7f0d33f3eb351abb6d5fe23b9513733ceef510